### PR TITLE
Uninstall NVT (removing versions from PyPI) before installing NVT in CI

### PR DESCRIPTION
**Pull request by karlhigley**

### CODE REVIEW

**Comment by albert17** [comment]
@jperez999 @karlhigley Could this be a problem if it is run while TF4Rec tests are running?

**Comment by karlhigley** [comment]
I think it should be okay most of the time, because the imports get loaded at the beginning and after that the files on disk don't matter. We might get an occasional flaky test run if things line up badly between NVT and Transformers4Rec, but I think we can rerun the tests in that case.

**Review by jperez999 — APPROVED** [review]

**Comment by jperez999** [comment]
rerun tests

File: ci/build_and_test.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ cd $nvt_directory
 
 echo "Installing NVTabular"
 python -m pip install --user --upgrade pip setuptools wheel pybind11 numpy==1.20.3
+python -m pip uninstall nvtabular
```
Comment (jperez999) [inline ci/build_and_test.sh:11]: can you add -y ?

**Pull request merged**